pkg/operator/config/manager/manager.yaml

```diff
@@ -149,6 +155,54 @@ spec:
         - name: LOG_LEVEL
           value: ${LOG_LEVEL}
 ---
+apiVersion: apps/v1
```

Comment:
so we're deploying the Windows DaemonSet even if the user doesn't want it (or will never use Windows containers). Maybe we can add a note on the quickstart that users can delete it afterwards? kubectl delete -n thundernetes-system daemonset nodeagent-win

Reply:
Ah yeah, the pods wouldn't get deployed but the daemon would stay there, I'll add that somewhere.